src/codec_aom.c

```diff
@@ -783,6 +783,9 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
         }
         minQuantizer = AVIF_CLAMP(minQuantizer, 0, 63);
         maxQuantizer = AVIF_CLAMP(maxQuantizer, 0, 63);
+        // Make sure that maxQuantizer >= quantizer >= minQuantizer.
```

Comment:
We can alternatively return an error if `maxQuantizer >= quantizer >= minQuantizer` does not hold. Do you think it is better to fix it up transparently?

Reply:
If this eventually leads to an error, we can also fail early with a more meaningful diagnostic message.